ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyMetadataFormSection.tsx

```diff
@@ -23,6 +23,14 @@ function PolicyMetadataFormSection(): ReactElement {
 
     return (
         <Form>
+            <Checkbox
+                label="Disable policy"
+                isChecked={values.disabled}
+                onChange={handleChange}
+                description="When checked, this policy will be disabled, stopping enforcement and alerts. By default, the policy is enabled to ensure continuous monitoring and protection."
```

Comment:
An observation, not necessarily a change request.

The prose has a **chiasm**, meaning **crossing over**, from Greek chi which has X shape.
* enforcement / monitoring
* alerts / protection

The correspondence seems more like:
* alerts / monitoring
* enforcement / protection

Reply:
I should've noted this was made up on the fly and @mansursyed will be providing updated text. Good comment nonetheless.